api/api_controller.go

```diff
@@ -2297,3 +2300,17 @@ func (c *Controller) ImportFromS3InventoryHandler() repositories.ImportFromS3Inv
 		})
 	})
 }
+
+func (c *Controller) ConfigGetConfigHandler() configop.GetConfigHandler {
+	return configop.GetConfigHandlerFunc(func(params configop.GetConfigParams, user *models.User) middleware.Responder {
+		c.setupRequest(user, params.HTTPRequest, []permissions.Permission{
```

Comment:
See how the other methods call `setupRequest`:
```
deps, err := c.setupRequest(...)
```
then handle the error, and use the returned `deps` rather than `c.deps`

Reply:
Works great! (Finally realized how to test it)